railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= camelized %></title>
+    <title><%= content_for(:title) || camelized %></title>
```

Comment:
This template gets evaluated before it's saved.
So:
```erb
<title><%= camelized %></title>
```
will generate something like the following in the erb file:
```html
<title>MyApp</title>
```

Your probably need to do something like:
```suggestion
    <title><%%= content_for(:title) %></title>
```

Reply:
Using `content_for` only makes sense if `:title` is also set in some of the views, which I think is a bit too opinionated.

Reply:
🤦 Sorry I wanted to say this... `<%%= content_for(:title) || "<%= camelized %>" %>`